misc/misc_test.go

```diff
@@ -0,0 +1,22 @@
+package misc
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRandString(t *testing.T) {
+	for i := 0; i < 100; i++ {
+		randStr1, err1 := RandString(10)
+		randStr2, err2 := RandString(10)
+
+		assert.Nil(t, err1)
+		assert.Nil(t, err2)
+
+		assert.Equal(t, 10, len(randStr1))
+		assert.Equal(t, 10, len(randStr2))
+
+		assert.NotEqual(t, randStr1, randStr2)
```

Comment:
nit: consider testing that all returned characters are valid (exists in `seed`).

Reply:
not nit, I would say this should be required.